### PR TITLE
[onert] Add shape inference for split op.

### PR DESCRIPTION
**Pull request by Hyunjun85**
Add shape inference for split operation and its tc.

ONE-DCO-1.0-Signed-off-by: H Kim <hyunjun2.kim@samsung.com>

### CODE REVIEW

File: runtime/onert/backend/cpu/KernelGenerator.cc [hunk 1]
```diff
@@ -921,7 +921,6 @@ void KernelGenerator::visit(const ir::operation::Split &node)
   const auto rank = node.param().rank;
   const auto axis = ops::getAxis(rank, node.param().axis, _current_op_seq_layout);
   auto axis_resolved = axis < 0 ? axis + rank : axis;
-  assert(0 <= axis_resolved && axis_resolved < rank);
```
Comment (chunseoklee) [inline runtime/onert/backend/cpu/KernelGenerator.cc:924]: Is this relevant to this PR ?

Comment (Hyunjun85) [reply runtime/onert/backend/cpu/KernelGenerator.cc:924]: Yes, when enter a dynamic tensor as input, the rank will be 0.
So, I move that condition into shape inf layer.

File: tests/nnapi/specs/V1_2/split_dynamic_float_nnfw.mod.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2018 The Android Open Source Project
```
Comment (hyunsik-yoon) [inline tests/nnapi/specs/V1_2/split_dynamic_float_nnfw.mod.py:2]: Please add samsung copyright.

File: runtime/onert/backend/cpu/KernelGenerator.cc (hunk 1 above)
Comment (Hyunjun85) [reply runtime/onert/backend/cpu/KernelGenerator.cc:924]: KernelGenerator::visit(const ir::operation::Split &node) called when compile time. So if input is a dynamic tensor, the it  couldn't get a intact input's shape. Even though,  "assert(0 <= axis_resolved && axis_resolved < rank)" tries to check rank of input. In this situation, It will ouccurr an assertion always.

File: tests/nnapi/specs/V1_2/split_dynamic_float_nnfw.mod.py (hunk 2 above)
Comment (Hyunjun85) [reply tests/nnapi/specs/V1_2/split_dynamic_float_nnfw.mod.py:2]: Ok. I applied.

**Comment by Hyunjun85** [comment]
Resolve merge conflict.
